notify/src/component/page-container/ArchiveDisplayCard.jsx

```diff
@@ -0,0 +1,52 @@
+import { useNotesContext } from "../../context/notes-context";
+import { MdOutlineUnarchive, MdDelete } from "react-icons/md";
+import { IoColorPaletteOutline } from "react-icons/io5";
+import "../note-display/noteDisplay.css";
+const ArchiveDisplayCard = () => {
+  const { notesState, notesDispatch } = useNotesContext();
+
+  return (
+    <>
+      <div className="note-display-container">
+        <h3>
+          {notesState.notes.filter(
```

Comment:
suggestion : You can destructure the notes in the useNotesContext function call itself that would avoid long chaining.